howto/stream/exchange-oob-data.md

```diff
@@ -24,83 +24,111 @@ The following instructions guide you to exchange OOB data using a specific imple
 
 The following instructions will walk you through how to set up data channels and perform data transmission in both directions between an Android application and a WebRTC platform.
 
-Before proceeding, ensure that you have set up a web-based streaming client using the Anbox Cloud streaming stack. If you haven't done so, please follow our setup [guide](https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/stream-client/).
-
 ### Prepare your web application
 
-In your web-based client application, import the {ref}`sec-streaming-sdk`.
+This guide builds upon the [streaming client setup tutorial](https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/stream-client/) for Anbox Cloud. Ensure you have completed the setup of a web-based streaming client as described in the tutorial till the [step where you set up stream client](https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/stream-client/#implement-the-stream-client).
+
+#### Extend `AnboxStream` Configuration
 
-Create a data channel (named `foo` in the following example) under the `dataChannels` property of an `AnboxStream` object and register event handlers that respond to the events sent from an Android application:
+Now, to create an `AnboxStream` object, add the dataChannels property to define a new data channel. For example:
```

Comment:
@adglkh Could you use backticks with dataChannels everywhere so that the documentation checks can pass?

dataChannels => `dataChannels`

Reply:
Sure, thanks for pointing this out